modules/ROOT/pages/datasense-concept.adoc

```diff
@@ -8,27 +8,27 @@ DataSense is a feature of Anypoint Studio that uses Mule event metadata to help
+Consider a flow in which you include an Anypoint Connector preceded or followed by a Transform Message component. The Mule runtime engine uses the working connection to the resource to retrieve and supply to DataWeave metadata about the payload and properties. This feature saves you the effort of manually inspecting Mule event structure.
 
-For example, imagine that you need to connect your organization's Salesforce account to Slack to notify your team about a new opportunity. To accomplish this, you can drop both Salesforce and Slack connectors on your Studio canvas and configure them to connect to your organization's accounts. Then, you can drop a Transform Message component between the connectors. Mule automatically captures the data type and structure information from each provider and prescribes the input and output for your data mapping. With this prescription in place, you can configure the mapping, populating the DataWeave code.
+For example, imagine that you must connect your organization's Salesforce account to Slack to notify your team about a new opportunity. To do this, you can drop both Salesforce and Slack connectors on your Studio canvas and configure them to connect to your organization's accounts. Then, you can drop a Transform Message component between the connectors. Mule automatically captures the data type and structure information from each provider and prescribes the input and output for your data mapping. With this prescription in place, you can configure the mapping, populating the DataWeave code.
 
 image::datasense-perceptive-flow-design-concept-cf42d.png[]
 
 == Use Datasense for Autocomplete
 
-Imagine that you are obtaining status update data from the Slack connector and you want to log the message you are sending to Slack, but you aren't familiar with the property names used by this connector. Instead of referring to Slack documentation, you can insert a Logger message processor after the Slack connector and begin to type in it, which activates autocomplete. For example, if you type `#[payload.` and then press `ctrl + space bar`, you get a list of all the properties and methods associated with that payload, including the properties returned by your request to the Slack connector.
+Imagine that you're obtaining status update data from the Slack connector. You want to log the message you're sending to Slack, but you aren't familiar with the property names used by this connector. Instead of referring to Slack documentation, you can insert a Logger message processor after the Slack connector and begin to type in it, which activates autocomplete. For example, if you type `#[payload.` and press `ctrl + space bar`, you get a list of all the properties and methods associated with that payload. In the list, you also get the properties returned by your request to the Slack connector.
```

Comment:
```suggestion
Imagine that you're obtaining status update data from the Slack connector. You want to log the message you're sending to Slack, but you aren't familiar with the property names used by this connector. Instead of referring to Slack documentation, you can insert a Logger message processor after the Slack connector and begin to type in it, which activates autocomplete. For example, if you type `#[payload.` and press `Ctrl+Spacebar`, you get a list of all the properties and methods associated with that payload. In the list, you also get the properties returned by your request to the Slack connector.
```

Reply:
OK! Thanks!